### DISCUSSION TITLE
Problema al Firmware Home Assistant

### DISCUSSION THREAD

**Discussion by Andrew186** [indomus/forum#1997] [Hardware & Firmware]
Salve a tutti, ho un problema da risolvere e da solo non riesco a venirne a capo.

Ultimamente sto riscontrando il problema, ossia dopo aver formattato e reinstallato HA sul mio raspberry funziona per qualche giorno ma poi quando tento di collegarmi dal browser in rete locale mi dice "impossibile raggiungere il sito" e non riesco ad collegarmi nemmeno da IP.

Voi avete avuto problemi simili? come avete qualche idea su come risolvere il problema?

Grazie

**Comment by Scialla** [comment]
Detto così ci vorrebbe una sfera di cristallo... magari ha semplicemente cambiato IP e tu cerchi di raggiungerlo a quello vecchio. Cerca di capire se è online, attacca un monitor al raspberry, guarda i log di HA, poi non hai nemmeno specificato quale versione di HA hai installato e come. Guarda i log, assegna IP statico al rasp. 
Senza Info, e senza sfera, darti una mano e impossibile

**Reply by Andrew186** [reply]
Il sistema operativo è il 9.2, installato tramite balenaEtcher su SD, l'IP è statico è la prima cosa che ho fatto, comunque ogni volta che entro da IP lo controllo prima di entrare.

Ultimamente, non so se sia collegata la cosa, quando cerco di aggiornare il core HA si "blocca" cioè non fa l'aggiornamento e quando tento per esempio di entrare in "dispositivi e servizi" mi da errore

**Reply by snakuzzo** [reply]
Devi dettagliare gli errori.
"Mi dà errore" non vuol dire assolutamente niente

**Reply by Andrew186** [reply]
Da questo errore "error while loading page profile" quando per esempio cerco di entrare nel profilo

**Reply by snakuzzo** [reply]
Non parlo degli errori che ti fa vedere sulla GUI, ma quelli presenti nei file di log di Core e del Supervisor

**Reply by Andrew186** [reply]
Il problema è che se non riesco accedere ad HA non riesco a visualizzare il file di log, almeno che non ci sia una possibilità di estrarlo dalla SD mettendola su un PC

**Reply by snakuzzo** [reply]
Hai letto cosa ti ha scritto @Scialla prima?

**Reply by Scialla** [reply]
per come guardare i log: https://www.home-assistant.io/hassio/commandline/

**Reply by Andrew186** [reply]
Mi devo procurare una micro HDMI per collegarlo ad uno schermo, e se provo collegarmi in ssh l'IP della raspberry con Putty non entra
